docs/roman/outlier_detection/main.rst

```diff
@@ -0,0 +1,38 @@
+Description
+===========
+
+:Classes: `romancal.outlier_detection.OutlierDetectionStep`
+:Aliases: outlier_detection, outlier_detection_scaled, outlier_detection_stack
+
+Processing multiple datasets together allows for the identification of bad pixels
+or cosmic-rays that remain in each of the input images, many times at levels which
+were not detectable by the :ref:`jump <jump_step>` step. The ``outlier_detection`` step
+implements the following algorithm to identify and flag any remaining cosmic-rays or
+other artifacts left over from previous calibrations:
+
+  - build a stack of input data
+
+    - all inputs will need to have the same WCS since outlier detection assumes
+      the same flux for each point on the sky, and variations from one image to
+      the next would indicate a problem with the detector during readout of that
+      pixel
```

Comment:
I'm not sure that I'd characterize a different distribution of CR as a detector problem.